NEWS

```diff
@@ -1,3 +1,65 @@
+KLEE 2.2, xx November 2020
+==========================
+
+Incorporating changes from 4 March 2020 to xx November 2020.
+Maintainers during this time span: @ccadar and @MartinNowack
+Documentation at http://klee.github.io/releases/docs/v2.2
+
+== Major features ==
+- Added support for C++ exceptions (@corrodedHash, @jbuening, @futile)
+- Correctly copy variadic arguments with byval attribute (@ccadar)
+- Modified the random path searcher to work on a subset of states (@kren1)
```

Comment:
I would count this more as a bugfix than a major feature

Reply:
I think any important bug fix is a feature ;)   In this case, it enables the use on RPS in more situations, which I think is an important feature.